package.json

```diff
@@ -31,14 +31,14 @@
     "coveralls": "^3.0.1",
     "eslint": "^4.18.1",
     "eslint-plugin-node": "^6.0.1",
-    "faucet": "^0.0.1",
+    "faucet": "0.0.1",
```

Comment:
Did we mean to pin this to a specific version?

Reply:
Not sure why npm changed that.